contrib/examples/actions/mistral-customfilters-json-escape.yaml

```diff
@@ -0,0 +1,11 @@
+---
+description: Example for using the custom filter "json_escape"
+enabled: true
+entry_point: workflows/tests/mistral-json-escape.yaml
+name: mistral-customfilters-json-escape
```

Comment:
Fixed in https://github.com/StackStorm/st2/pull/3565/commits/821ddb2b24f2fd4e1c5537bd2fb1010e2b31e730